README.md

```diff
@@ -247,6 +247,80 @@ in {
 
 </details>
 
+<details><summary><strong>Example flake</strong> (using overlays)</summary>
+
+If you pass your flake inputs to `specialArgs`, you can apply the overlay within your `configuration.nix`:
+
+```nix
+# flake.nix
+{
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
+    umu.url = "github:Open-Wine-Components/umu-launcher?dir=packaging/nix";
+    umu.inputs.nixpkgs.follows = "nixpkgs";
```

Comment:
Think we should leave a note here about stable channels missing packages. They won't be in a stable release for 2 months and future dependencies may also be missing.

Reply:
Isn't that already covered by [L167](https://github.com/Open-Wine-Components/umu-launcher/blob/8bf01dd9ba5dd180a963776549acdc20265560e3/README.md?plain=1#L167) and [L195](https://github.com/Open-Wine-Components/umu-launcher/blob/8bf01dd9ba5dd180a963776549acdc20265560e3/README.md?plain=1#L195)?

Also, the `follows` is less relevant when using overlays, as umu's nixpkgs input will only be used for `nixpkgs.lib` in that scenario.